NuGet.Docs/Release-Notes/NuGet-3.2.md

```diff
@@ -0,0 +1,104 @@
+* Now retrying network connections every 200ms for a maximum of 5 attempts in the case of HTTP 5xx failures - [1120](https://github.com/NuGet/Home/issues/1120) 
+* Improved handling of server redirect responses during a push command - [1051](https://github.com/NuGet/Home/issues/1051)
+* `nuget install -source` now supports either URL or repository name from nuget.config as an argument - [1046](https://github.com/NuGet/Home/issues/1046)
+* Missing packages that were not located on a repository during a restore are now reported as errors instead of warnings [1038](https://github.com/NuGet/Home/issues/1038)
+* Corrected multipartwebrequest handling of \r\n for Unix/Linux scenarios - [776](https://github.com/NuGet/Home/issues/776)
+
+There are a number of fixes to issues with various commands:
+
+* Push command no longer does a GET before a PUT against a package source - [1237](https://github.com/NuGet/Home/issues/1237)
+* List command no longer repeats version numbers - [1185](https://github.com/NuGet/Home/issues/1185)
+* Pack with the -build argument now properly supports C# 6.0 - [1107](https://github.com/NuGet/Home/issues/1107)
+* Corrected issues attempting to pack an F# project built with Visual Studio 2015 - [1048](https://github.com/NuGet/Home/issues/1048)
+* Restore now no-ops when packages already exist - [1040](https://github.com/NuGet/Home/issues/1040)
+* Improved error messages when packages.config file is malformed - [1034](https://github.com/NuGet/Home/issues/1034)
+* Corrected restore command with -SolutionDirectory switch to work with relative paths - [992](https://github.com/NuGet/Home/issues/992)
+* Improved Updated command to support solution-wide update - [924](https://github.com/NuGet/Home/issues/924)
+
+A complete list of issues addressed in this release can be found in the NuGet GitHub [Command-Line milestone](https://github.com/nuget/home/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A3.2.0-commandline+is%3Aclosed+-label%3AClosedAs%3ADuplicate).
+
+## Visual Studio extension updates
+
+### New Features in Visual Studio
+
+* A new context menu item was added to the Solution Explorer on the solution node that allows packages to be restored without building the solution ([1274](https://github.com/NuGet/Home/issues/1274)).
+
+![New 'Restore Packages' Context Menu Item](images/NuGet-3.2/newContextMenu.png)
+
+### Updates and Fixes in Visual Studio
+
+The fixes for authenticated feeds were rolled up and addressed in the extension as well.  The following authentication items were also addressed in the extension:
+
+* Now correctly treating NuGet v3 authenticated feeds properly, instead of as v2 authenticated feeds - [1216](https://github.com/NuGet/Home/issues/1216)
+* Corrected request for authentication credentials in projects using project.json and communicating with v2 feeds - [1082](https://github.com/NuGet/Home/issues/1082)
+
```

Comment:
make it look like a title (either H4/H5 or bold/underline), same for the rest of the items